memiavl/multitree.go

```diff
@@ -88,7 +88,7 @@ func LoadMultiTree(dir string, zeroCopy bool, cacheSize int) (*MultiTree, error)
 		treeMap[name] = NewFromSnapshot(snapshot, zeroCopy, cacheSize)
 	}
 
-	sort.Strings(treeNames)
+	slices.Sort(treeNames)
```

Comment:
golang advice: consider using the newer slices.Sort function, which runs faster.